public/app/core/components/NestedFolderPicker/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface RootFolder {
```

Comment:
This is probably a bad name. This structure is probably available for the children too.